### ISSUE TITLE
Pretrained model for all-synthetic for Virtual_Kitti-Kitti dataset

### ISSUE THREAD

**Issue by koutilya-pnvr** [lyndonzheng/Synthetic2Realistic#35]
Hi,

It will be great if you can share the pretrained depth estimator model on 'all-synthetic' for the outdoor dataset. I am working on that specific model which I marked from the results section of the paper.
<img width="1044" alt="Screen Shot 2019-10-02 at 11 50 14 PM" src="https://user-images.githubusercontent.com/24998956/66098362-477d4280-e570-11e9-80b4-d49b8c016aca.png">

Thanks a lot in advance.

**Comment by koutilya-pnvr** [comment]
It will be great if you can share the training details at least. I tried to replicate the 'all-synthetic' lower bound performance and got the following results:

all-synthetic results:
   abs_rel,      sq_rel,      rmse,    rmse_log,  err < 1.25,  err < 1.25^2,  err < 1.25^3
    0.2671,    2.4044,    6.5776,    0.4046,      0.5421,        0.7768,           0.8881

Depth Estimator: _UNetGenerator() from this repo
Dataset: All Virtual Kitti Synthetic Data samples
Loss: L1 Loss between the predicted Syn depth and GT depth
Optimizer: Adam, 1e-4 learning rate

The performance metrics differs hugely. Just wondering if I am missing something during implementation.

**Comment by lyndonzheng** [comment]
@koutilya40192 The all synthetic model uses the _UNetGenerator() as task network, and only use the vKitti data for training. The RGB as Input, and directlty predict the depth. You training method is close to us. Do you clip the depth range for the testing? Following the previous work, we clip the depth to [1,50] for the final testing.

**Comment by koutilya-pnvr** [comment]
I understand that the baseline was evaluated with dropout layers in the UNet, since vKitti is a limited data which could potentially overfit the UNet. Since I was using the same UNet from the T2Net, I didn't use any dropout layers, which could be the reason for a poor baseline. Thanks a lot @lyndonzheng for sharing this useful information over e-mails.
